nuitka/nodes/BuiltinAnyNodes.py

```diff
@@ -0,0 +1,56 @@
+#     Copyright 2019, Kay Hayen, mailto:kay.hayen@gmail.com
+#
+#     Part of "Nuitka", an optimizing Python compiler that is compatible and
+#     integrates with CPython, but also works on its own.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+#
+""" Node for the calls to the 'any' built-in.
+
+"""
+
+from nuitka.specs import BuiltinParameterSpecs
+
+from .ExpressionBases import ExpressionBuiltinSingleArgBase
+from .shapes.BuiltinTypeShapes import ShapeTypeBool
+
+
+class ExpressionBuiltinAny(ExpressionBuiltinSingleArgBase):
+    kind = "EXPRESSION_BUILTIN_ANY"
+
+    builtin_spec = BuiltinParameterSpecs.builtin_any_spec
+
+    def getIntegerValue(self):
+        value = self.getValue()
+
+        if value.hasShapeSlotAny():
+            return value.getIterationLength()
+        else:
+            return None
+
+    def computeExpression(self, trace_collection):
+        return self.getValue().computeExpressionAny(
```

Comment:
The `computeExpressionAny` would be what is missing. You can see something similar at:

https://github.com/Nuitka/Nuitka/blob/cb0e0beb72041adeab1065306d5413a6bd868611/nuitka/nodes/ExpressionBases.py#L345

Reply:
Actually I was sort of misleading you here. The slot concept in Nuitka is the one of Python. There are slots for `__float__`, `__int__`, and there is `__len__` for types to overload. For any, I do not think that is the case necessarily, although I would have to check. 

My suspect is that is uses mere iteration. We would have to check the implementation in Python. I am going to send you a link for what code to look at, once I made a move of a repository not currently under Nuitka organisation, although it should be. So allow a few hours for that.

Reply:
@kayhayen You are right. Both `len` and `any` use iteration as input. Still I have created the slot for `any`.